src/main/java/org/spongepowered/common/data/manipulator/mutable/SpongeDyeableData.java

```diff
@@ -36,7 +34,7 @@
 public class SpongeDyeableData extends AbstractSingleCatalogData<DyeColor, DyeableData, ImmutableDyeableData> implements DyeableData {
 
     public SpongeDyeableData(DyeColor dyeColor) {
-        super(DyeableData.class, checkNotNull(dyeColor), Keys.DYE_COLOR, ImmutableDyeableData.class);
+        super(DyeableData.class, dyeColor, Keys.DYE_COLOR, ImmutableDyeableData.class);
```

Comment:
You'll want to add `@ImplementationRequiredForTest` to the class, which prevents the ManipulatorTest from running (since the dye colors aren't available in the testing environment).